dnf-behave-tests/fixtures/specs/dnf-transaction/wilee-1.spec

```diff
@@ -0,0 +1,28 @@
+Name: wilee
+Version: 1.0
+Release: 1.fc29
+Summary: Package that adds a plugin that removes package
+
+License: GPLv3+
+Url: None
+
+%description
+This package adds a plugin that removes the package that installed it
+
+%install
+mkdir -p %{buildroot}/dnf-plugins/
+cat >> %{buildroot}/dnf-plugins/wilee-dnf-plugin.py <<HERE
+import dnf
+
+class Wilee(dnf.Plugin):
+    name = "wilee"
+
+    def resolved(self):
+        print("RESOLVED")
+        os.system("rpm -i wilee")
```

Comment:
This seems to be doing an install, not a remove :slightly_smiling_face:

Reply:
Fixed: it was a spelling issue :)